multi-account-private-apigw/centralAccount/src/cfnresponse.py

```diff
@@ -0,0 +1,54 @@
+#  Copyright 2016 Amazon Web Services, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Are you sure the license text is still accurate? If so, update the date

Reply:
You could even remove the whole paragraph as the repo already has a license attached to it